pkg/codegen/template.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+const objectsFromRequestTemplate = `
+package {{ .package }}
+
+import (
+	{{ range .types }}
+	"{{ .Package }}"{{ end }}
+	"github.com/rancher/wrangler/pkg/webhook"
+	admissionv1 "k8s.io/api/admission/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+{{ range .types }}
+
+func {{ .Name }}OldAndNewFromRequest(request *webhook.Request) ({{ .Type }}, {{ .Type }}, error) {
```

Comment:
Add a comment above each function to describe what it does

Reply:
Done.